sig-architecture/production-readiness.md

```diff
@@ -80,9 +80,28 @@ To become a reviewer:
  * Read/study previous PRR comments and production readiness responses in existing KEPs.
  * Choose some KEPs requiring PRR and perform a review. Put "shadow prod readiness review"
    in your review comments so that the assigned PRR approver knows your intent.
- * After at least one release cycle, if you have shown good judgement and quality reviews,
-   you can propose yourself as approver by submitting a PRR to add your GitHub
-   handle to the `prod-readiness-approvers` alias in [OWNERS_ALIASES].
+
+### Becoming an approver
+
+After serving as reviewer/shadow for at least one release and showing good judgement and quality reviews,
```

Comment:
this is on the eyes of existing PRR reviewers? "showing good judgement and quality reviews"

Reply:
ok, Joe asked it here https://github.com/kubernetes/community/pull/7305/files#r1198310088 you can resolve this comment